migrations/2023-06-21-153242_add_captcha/up.sql

```diff
@@ -0,0 +1,5 @@
+create table captcha_answer (
+    uuid text not null primary key,
```

Comment:
You shouldnt store uuids as string, but use the [Uuid](https://docs.diesel.rs/2.1.x/diesel/pg/sql_types/struct.Uuid.html) type. This requires enabling the `uuid` feature for diesel. 

I believe this is the problem dessalines referred to.

Reply:
thanks, should be done now